src/Constants.java

```diff
@@ -13,4 +13,6 @@
     public static final int     STADIUM_MARGIN_RIGHT = 695;
     public static final double  GUARD_SPAWN_RATE = .98;
     public static final int     CHARACTER_VELOCITY = 550;
+	public static final int JUMP_TIME = 1000;
```

Comment:
Identation

Reply:
This is the only indentation issue that appears on my local level in vim. I'm not sure what you want changed about the rest of them.

Reply:
I didn't change any code. All of the code you are trying to check is indented incorrectly; I don't know how it looks on your end but it should be changed so it's right on github